controllers/profiles/getProfileByName.js

```diff
@@ -27,27 +27,29 @@ module.exports = async (req, res) => {
       }
     });
 
-    if (!user)
+    if (!user || user?.dataValues)
       return res.json({
         code: 404,
         message: `No user with username ${username} found`,
         status: 'error'
       });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition in the if statement seems to be incorrect. It will return an error message even when a user is found because `user?.dataValues` will be truthy when a user is found. You should check for the absence of `user?.dataValues` instead.

```diff
-    if (!user || user?.dataValues)
+    if (!user || !user?.dataValues)
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
mas @fajarism untuk yang ini sepertinya memang miss ya ?

Reply:
mas @fajarism done on latest commit mas, kelewatan dikit